src/main/scala/com/github/mrpowers/spark/spec/ml/classification/IrisKMeansClustering.scala

```diff
@@ -0,0 +1,41 @@
+package com.github.mrpowers.spark.spec.ml
+
+import com.github.mrpowers.spark.spec.sql.SparkSessionWrapper
+import org.apache.spark.ml.feature.VectorAssembler
+import org.apache.spark.sql.DataFrame
+import org.apache.spark.ml.clustering.{KMeans, KMeansModel}
+
+object IrisKMeansClustering
+  extends SparkSessionWrapper {
+
+  val IrisDF = spark
+    .read
+    .option("header", "true")
+    .option("inferSchema", "true")
+    .csv("./src/test/resources/iris/iris.csv")
```

Comment:
Should we bifurcate this CSV into one file that's used to train the model and another file that's used to evaluate the accuracy of our model?  @nathanlim45 - thoughts?